src/Extensions/Logging/Internal/Replayable/LogMessageInformation.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Omex.Extensions.Logging.Replayable
+{
+	internal struct LogMessageInformation
+	{
+		public string Category { get; }
+		public EventId EventId { get; }
+		public int ThreadId { get; }
+		public string Message { get; }
+
+
+		public LogMessageInformation(string category, EventId eventId, int threadId, string message)
```

Comment:
Consider validating string

Reply:
This class would be used only internally inside of the assembly, I don't think it requires validation